core/arch/arm/kernel/secure_partition.c

```diff
@@ -14,8 +14,9 @@
 #include <kernel/ts_store.h>
 #include <ldelf.h>
 #include <mm/core_mmu.h>
-#include <mm/fobj.h>
 #include <mm/mobj.h>
+#include <mm/fobj.h>
```

Comment:
Unrelated changes for this commit?
By the way, please keep these in alphabetic order.

Reply:
Done